docs/src/reference/the-graph.asciidoc

```diff
@@ -404,17 +405,18 @@ applications outside of TinkerPop, GraphML may be the best choice to do that. Co
 * Produce a subgraph and export it to GraphML to be consumed by and visualized in link:https://gephi.org/[Gephi].
 * Migrate the data of an entire graph to a different graph database not supported by TinkerPop.
 
-As GraphML is a specification for the serialization of an entire graph and not the individual elements of a graph,
-methods that support input and output of single vertices, edges, etc. are not supported.
+As GraphML is a specification for the serialization of an entire graph and not the individual elements of a graph.
```

Comment:
This change reads like it made this sentence incomplete.  I guess you could remove the word "As" from the front to make it read as complete.